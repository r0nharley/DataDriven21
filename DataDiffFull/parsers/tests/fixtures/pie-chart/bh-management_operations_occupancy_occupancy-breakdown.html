<svg version="1.1" class="highcharts-root" style="font-family:&quot;Lucida Grande&quot;, &quot;Lucida Sans Unicode&quot;, Arial, Helvetica, sans-serif;font-size:12px;" xmlns="http://www.w3.org/2000/svg" width="344" height="423">
    <desc>Created with Highstock 5.0.6</desc>
    <defs>
        <clipPath id="highcharts-k7uut2n-47">
            <rect x="0" y="0" width="324" height="345" fill="none"></rect>
        </clipPath>
    </defs>
    <rect fill="#ffffff" class="highcharts-background" x="0" y="0" width="344" height="423" rx="0" ry="0"></rect>
    <rect fill="none" class="highcharts-plot-background" x="10" y="10" width="324" height="345"></rect>
    <rect fill="none" class="highcharts-plot-border" x="10" y="10" width="324" height="345"></rect>
    <g class="highcharts-series-group">
        <g class="highcharts-series highcharts-series-0 highcharts-pie-series highcharts-color-undefined highcharts-tracker highcharts-series-hover" transform="translate(10,10) scale(1 1)">
            <path fill="#6eda55" d="M 303.62122779444684 93.83979508040042 A 162 162 0 1 1 161.7749852500324 10.500156270562115 L 161.78887504941312 20.50014662423112 A 152 152 0 1 0 294.87917669602416 98.69536328531397 Z" class="highcharts-halo highcharts-color-3" fill-opacity="0.25"></path>
            <path fill="#000000" d="M 161.96904167303833 21.00000315269085 A 152 152 0 0 1 169.41409859310244 21.180926290364425 L 164.22422957793074 127.45427788710933 A 45.6 45.6 0 0 0 161.9907125019115 127.40000094580725 Z" class="highcharts-point highcharts-color-0 " stroke="#ffffff" stroke-width="1" stroke-linejoin="round"></path>
            <path fill="#fc7570" d="M 169.5659139344599 21.188416297252388 A 152 152 0 0 1 241.29242482107588 43.3207365613375 L 185.78772744632278 134.09622096840124 A 45.6 45.6 0 0 0 164.26977418033798 127.45652488917571 Z" class="highcharts-point highcharts-color-1 " stroke="#ffffff" stroke-width="1" stroke-linejoin="round"></path>
            <path fill="#9b9bd7" d="M 241.4220644166922 43.40009381256945 A 152 152 0 0 1 294.80530563202745 99.06252103307976 L 201.84159168960824 150.8187563099239 A 45.6 45.6 0 0 0 185.82661932500767 134.12002814377084 Z" class="highcharts-point highcharts-color-2 " stroke="#ffffff" stroke-width="1" stroke-linejoin="round"></path>
            <path fill="rgb(135,243,110)" d="M 294.87917669602416 99.19536328531397 A 152 152 0 1 1 161.78887504941312 21.00014662423112 L 161.93666251482392 127.40004398726933 A 45.6 45.6 0 1 0 201.86375300880727 150.8586089855942 Z" class="highcharts-point highcharts-color-3 highcharts-point-hover" stroke="#ffffff" stroke-width="1" stroke-linejoin="round"></path>
        </g>
        <g class="highcharts-markers highcharts-series-0 highcharts-pie-series highcharts-color-undefined highcharts-series-hover" transform="translate(10,10) scale(1 1)"></g>
    </g>
    <g class="highcharts-data-labels highcharts-series-0 highcharts-pie-series highcharts-color-undefined highcharts-tracker" transform="translate(10,10) scale(1 1)" opacity="1">
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-1 highcharts-tracker" style="" transform="translate(182,51)"></g>
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-2 highcharts-tracker" style="" transform="translate(231,82)"></g>
        <g class="highcharts-label highcharts-data-label highcharts-data-label-color-3 highcharts-tracker" style="" transform="translate(71,263)"></g>
    </g>
    <g class="highcharts-legend" transform="translate(23,367)">
        <rect fill="none" class="highcharts-legend-box" rx="0" ry="0" x="0" y="0" width="298" height="41" visibility="visible"></rect>
        <g>
            <g>
                <g class="highcharts-legend-item highcharts-pie-series highcharts-color-0" transform="translate(8,3)">
                    <text x="21" style="color:#333333;font-size:12px;font-weight:normal;cursor:pointer;fill:#333333;" text-anchor="start" y="15">
                        <tspan>Admin/Down</tspan>
                    </text>
                    <rect x="2" y="4" width="12" height="12" fill="#000000" rx="6" ry="6" class="highcharts-point"></rect>
                </g>
                <g class="highcharts-legend-item highcharts-pie-series highcharts-color-1" transform="translate(130.78125,3)">
                    <text x="21" y="15" style="color:#333333;font-size:12px;font-weight:normal;cursor:pointer;fill:#333333;" text-anchor="start">
                        <tspan>Vacant</tspan>
                    </text>
                    <rect x="2" y="4" width="12" height="12" fill="#fc7570" rx="6" ry="6" class="highcharts-point"></rect>
                </g>
                <g class="highcharts-legend-item highcharts-pie-series highcharts-color-2" transform="translate(213.453125,3)">
                    <text x="21" y="15" style="color:#333333;font-size:12px;font-weight:normal;cursor:pointer;fill:#333333;" text-anchor="start">
                        <tspan>Occ-NTV</tspan>
                    </text>
                    <rect x="2" y="4" width="12" height="12" fill="#9b9bd7" rx="6" ry="6" class="highcharts-point"></rect>
                </g>
                <g class="highcharts-legend-item highcharts-pie-series highcharts-color-3" transform="translate(8,18)">
                    <text x="21" y="15" style="color:#333333;font-size:12px;font-weight:normal;cursor:pointer;fill:#333333;" text-anchor="start">
                        <tspan>Occ-No NTV</tspan>
                    </text>
                    <rect x="2" y="4" width="12" height="12" fill="#6eda55" rx="6" ry="6" class="highcharts-point"></rect>
                </g>
            </g>
        </g>
    </g>
    <g class="highcharts-label highcharts-tooltip highcharts-color-3" style="cursor:default;pointer-events:none;white-space:nowrap;" transform="translate(54,34)" opacity="1" visibility="visible">
        <path fill="rgba(255, 255, 255, 0)" class="highcharts-label-box highcharts-tooltip-box" d="M 0 0 L 123 0 C 123 0 123 0 123 0 L 123 102 C 123 102 123 102 123 102 L 0 102 C 0 102 0 102 0 102 L 0 0 C 0 0 0 0 0 0"></path>
    </g>
</svg>