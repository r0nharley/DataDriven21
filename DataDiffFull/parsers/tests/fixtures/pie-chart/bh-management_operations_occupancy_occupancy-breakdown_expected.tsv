Admin/Down	Percentage	0.779859221151
Vacant	Percentage	7.94183166029
Occ-NTV	Percentage	8.16464858061
Occ-No NTV	Percentage	83.0470493054